.commitlintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+    "extends": [
+        "@commitlint/config-conventional"
+    ]
+}
```

Comment:
Nift pick: configure your editor to add a new line to all files.

Reply:
Oh, thanks, i didn't know that was an option!